### ISSUE TITLE
[BUG] MA2 (Console) Cant control Companion

### ISSUE THREAD

**Issue by BSVTFrankyx59** [bitfocus/companion#2560]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Hello,
I need for a production that I can control with the grandma2 Companion, but unfortunately I have found no real information on the internet and also no instructions. I can control the console via companion but unfortunately not the other way around. Neither MSC nor telnet work. I had tried it via the ArtNet Listener but unfortunately I get absolutely no signal in there. I had already checked the network configuration several times but there is no error. If someone has a clue what it can be or can write me in detail what I have to do that I can control companion via the grandma2. 

### Environment (please complete the following information)

```markdown
- OS: Windows 10
- Browser: Firefox
- Companion Version: 3.0.1
```

**Comment by istnv** [comment]
MSC control of Companion is not supported.
Telnet should work via the TCP port. Do you have the TCP listener turned on?
![image](https://github.com/bitfocus/companion/assets/51003420/d5e2037b-707f-433b-8554-247f6e7ce462)

**Comment by Julusian** [comment]
correct, it is fixed in the betas not 3.0.1. Have you tried a recent beta?

**Comment by Julusian** [comment]
There have been fixes in the betas for the artnet listener since 3.0.1, so I expect this is fixed there and will be in the next release
